vaadin-notification-flow-parent/vaadin-notification-flow/pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>flow-test-util</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-html-components</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Seems this is not needed anymore after removing the unused import.

Reply:
Thanks, removed.